### ISSUE TITLE
Single chain config doesn't work

### ISSUE THREAD

**Issue by uvizhe** [family/connectkit#342]
When I configure wagmi with a single chain, e.g., `goerli` I get the following modal when I connect my wallet:

![Screenshot from 2024-02-05 10-38-57](https://github.com/family/connectkit/assets/1889712/30c3ae04-28d9-49a8-83c9-06e58496487d)

I believe this is due to an incorrect condition in this code:
https://github.com/family/connectkit/blob/1ccc9f11c67226b7a24cfd22a215b15ac70a7590/packages/connectkit/src/components/ConnectKit.tsx#L208

I guess it have to be this instead (and I tested it): `Boolean(chain && !chains.some((x) => x.id === chain?.id))`

**Comment by lochie** [comment]
Sorry about this one. We have an incoming patch on it's way
https://github.com/family/connectkit/pull/341

**Comment by lochie** [comment]
@uvizhe if you need a temporary fix while you wait, you can disable this Switch Networks popup by setting `enforceSupportedChains` to `false`.
https://docs.family.co/connectkit/api-reference#enforceSupportedChains

Example:
```jsx
<ConnectKitProvider options={{
  enforceSupportedChains: false
}}>
```

**Comment by uvizhe** [comment]
Oh, sorry for duplicate, I didn't notice a prior issue.

**Comment by lochie** [comment]
This fix is now available in ConnectKit `1.7.1`. Thanks again for alerting us to this issue 🫡
